core/wiki/macros/toc.tid

```diff
@@ -1,218 +1,422 @@
 title: $:/core/macros/toc
 tags: $:/tags/Macro
 
+<!--
+Naming rules / pattern
+- functions: tf.toc-variableName, "tf." -> function prefix reservered for TW, "toc-" -> global macro prefix, "variableName"
+- procedures eg: toc-procedureName,
+- macro parameters: path, itemTemplate
+- derived macro parameters: _path, _itemTemplate -> derived variables start with an underscore
 
-\define toc-open-icon() $:/core/images/down-arrow
-\define toc-closed-icon() $:/core/images/right-arrow
+The toc macro code gets more and more complex from top to bottom.
+toc-tabbed-internal-nav calls toc-tabbed-external-nav, which callse toc-selective-expandable, which calls toc-expandable
+
+The very first toc macro is _not_ "reused". So it can be used as a "simple" reverence implementation,
```

Comment:
The use of double quotes is unusual and a bit clumsy. What are they trying to accomplish?

Reply:
They give me a visual hint about the imortant words in the text, since comments cannot be formatted.